sisl/geom/bilayer.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+
+from sisl import Atom, geom, Cuboid
+
+__all__ = ['bilayer']
+
+
+def bilayer(bond=1.42, bottom_atom=None, top_atom=None, stacking='AB', twist=(0, 0), separation=3.35, ret_angle=False, layer='both'):
+    """ Commensurate unit cell of a hexagonal bilayer structure, possibly with a twist angle.
+
+    This routine follows the prescription of twisted bilayer graphene found in
+    Laissardiere et al., Nano Lett. 10, 804-808 (2010).
+
+
+    Parameters
+    ----------
+    bond : 1.42
+       bond length (in Angstrom) between atoms in the honeycomb lattice
+    bottom_atom : Atom(6)
+       atom (or atoms) in the bottom layer
+    top_atom : Atom(6)
+       atom (or atoms) in the top layer
+    stacking : {'AB', 'AA'}
+       stacking sequence of the bilayer
+    twist : (0, 0)
+       integer coordinates (m, n) defining a commensurate twist angle
+    separation : float
+       distance between the two layers (in Angstrom)
+    ret_angle : bool
+       return the twist angle (in degrees) in addition to the geometry instance
+    layer : {'both', 'bottom', 'top'}
+       control which layer(s) to include in geometry
+    """
+    if bottom_atom is None:
+        bottom_atom = Atom(Z=6, R=bond * 1.01)
+    if top_atom is None:
```

Comment:
Wouldn't it make sense to do:
```python
if top_atom is None:
    top_atom = bottom_atom
```
?

Reply:
I get the point, but what if the user only specifies `top_atom`? Maybe one should rather get two identical layers, if only one of the two `atom` parameters is given?

Reply:
Agreed, good idea.